src/clr-addons/themes/phs/css.overrides.scss

```diff
@@ -63,6 +63,84 @@
     }
   }
 
+  .datagrid-row:not(
+      .datagrid-selected,
+      .datagrid-highlight-error,
+      .datagrid-highlight-success,
+      .datagrid-highlight-info,
+      .datagrid-highlight-warning
+    ) {
+    background-color: $clr-highlight-neutral-50;
+
+    .datagrid-select {
+      background-color: $clr-highlight-neutral-50;
+    }
+  }
+
+  .datagrid-row:not(
+      .datagrid-selected,
+      .datagrid-highlight-error,
+      .datagrid-highlight-success,
+      .datagrid-highlight-info,
+      .datagrid-highlight-warning
+    ):hover {
+    background-color: $clr-highlight-neutral-200;
+
+    .datagrid-select {
+      background-color: $clr-highlight-neutral-200;
+    }
+  }
+
+  .btn-success {
+    background-color: $clr-highlight-success-700;
```

Comment:
instead of overriding css for that, we could just rewrite the css3 variables already in use by the buttons.
e.g. btn-success uses the variable --clr-btn-success-bg-color, which should be overwritten by the PHS theme instead